install_scripts/templates/common/kubernetes.sh

```diff
@@ -1,4 +1,12 @@
 
+UBUNTU_1604_K8S_9=ubuntu-1604
```

Comment:
Isn't this `ubuntu-1604-v1.9.3-${DATE}`? That's what's in the dockerfile, at least

Reply:
Yeah, I'll clean this up.